src/main/java/org/terracotta/statistics/derived/EventParameterSimpleMovingAverage.java

```diff
@@ -58,15 +61,15 @@ public Double value() {
   }
 
   public ValueStatistic<Double> averageStatistic() {
-    return this::average;
+    return supply(GAUGE, this::average);
```

Comment:
`gauge(this::average)` reads a little nicer.

Reply:
I'll change all 👍